src/reports/components/report-sections/passed-checks-section.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+.passed-checks-container {
+    padding-bottom: 58px;
```

Comment:
It feels a little off for the 3 individual sections to be identical except for this padding.

For the identical parts, did you already explore adjusting the CSS of the common inner components? 

For the different paddings, the individual section components aren't responsible for the order in which the sections appear (the composing component above them is), but this bit of CSS is relies on the "passed" section going last; would it be possible to instead add top-margin or top-padding to the h2-level section headings?

Reply:
(the latter might also remove the need to repeat this padding in `automated-checks-not-run-container`)